_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "What DevOps Looks Like In Terms Of Our Process"
+ref: what-devops-looks-like
+author: "Omar Nasr, in collaboration with the SDS and the IT Strategy team"
+lang: en
+date: "2020-02-16"
+last_modified: "2020-02-16"
+excerpt_separator: <!--more-->
+---
+
+<!--markdownlint-disable MD033-->
+
+There has been a large shift in our job as the technology organization of ESDC. We are now, more than ever, required to respond exponential change in technology and requirements faster, cheaper and with higher quality. Gone are the days where we can plan projects 5-10 years in advance. We hear the term DevOps being used as a new paradigm for software delivery. However, it is difficult to picture how this actually looks in our organization in terms of a tangible item such as the release process.
+<!--more-->
+
+Large IT projects pose a significant risk to our organization. We've seen it over and over, by the time these projects are completed, we have expended significant resources to produce a product that no longer meets our client's needs and are forced to spend even more to catch up to changed requirements. Continuing with this operating model means it is becoming more and more difficult to respond to the increasing social change and needs of our citizens in a [digital age](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html). Fortunately, we are not a unique case, many technology organizations around the world are facing similar issues in both the public and private domain. A lot of organizations have responded to this by adopting DevOps methodologies, techniques, tools and processes. You may have heard your co-worker or manager talking about the need to apply DevOps principles. However, to address the elephant in the room, how can this possibly be achieved with the large number of processes currently in place. What does DevOps actually look like in terms of our organization? More specifically, how can we incorporate these principles in our release process to be able to enable frequent releases, automated testing and checks, and be able to quickly respond to changes. What work needs to be done to shift to this model?
```

Comment:
Wondering if the 'to address the elephant in the room' adds value? Sentence would be tighter as "But can this possibly be achieved with the large number of processes currently in place?".